### PR TITLE
Better user/applications documentation

### PR DESCRIPTION
**Pull request by tribly**
Since only the user who created the application is allowed to get notifications from it, this should be clearer. 
There are no errors thrown, or warnings given when you send notifications and your android can't receive them.

### CODE REVIEW

**Comment by eternal-flame-AD** [comment]
Thanks! I think this piece of information should be also documented in `intro.md`. Can you consider including this in the introduction as well? :)

**Comment by tribly** [comment]
I am not really sure what `intro.md` you mean, I can't find any.

**Comment by eternal-flame-AD** [comment]
This file, https://github.com/gotify/website/blob/master/docs/index.md

`index.md`. Sorry for the mistake.

**Comment by tribly** [comment]
Took the lounge route, sorry :D Hope it's acceptable. Not 100% sure if the change from client -> user in the first paragraph is correct though.

**Comment by eternal-flame-AD** [comment]
Oh I am afraid this is not the correct explanation. I could have explained my idea clearer, my fault. Please revert this replacement , just append a sentence after the client and application definition, ex:

A user is owns its own collection of clients and applications, and clients only gets messages from the applications owned by the same server.

This way there would be of no way people could miss it:)

**Comment by tribly** [comment]
Alrighty, hope that's fine :)

File: docs/pushmsg.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: pushmsg
 title: Push messages
 ---
 
-As already indicated in [Intro](index.md) you need an application to push messages to gotify/server.
+As already indicated in [Intro](index.md) you need an application to push messages to gotify/server. Only the user who created the application is able to get its notifications.
```
Comment (jmattheis) [inline docs/pushmsg.md]: In gotify/server we normally use the term message. Clients then can send notifications for received messages.
```suggestion
As already indicated in [Intro](index.md) you need an application to push messages to gotify/server. Only the user who created the application is able to see its messages.
```

File: docs/index.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ However a client is not allowed to send messages.
 
 An application is a device or application that only can send messages.
 
+A user owns its own collection of clients and applications, and clients only gets messages from the applications owned by the same user.
```
Comment (jmattheis) [inline docs/index.md]: `Owns its own` doesn't sound so good IMO how about:
```suggestion
Users are only able to manage (view/edit/delete) clients and applications (includes messages sent by the app) that they created.
```

**Comment by tribly** [comment]
I'm fine with whatever, just want to make sure people see directly how users and apps correlate.

**Pull request closed without merging**